data/remnant missions.txt

```diff
@@ -216,7 +290,11 @@ mission "Remnant: Defense 3"
 		government "Korath"
 		personality heroic nemesis staying
 		system "Parca"
-		fleet "Korath Raid" 2
+		fleet
+			names "korath"
+			cargo 3
+			variant
+				"Korath Chaser" 10
```

Comment:
Because I want non-veteran players to be able to at least get to the point where they can buy basic Remnant ships without needing to have already purchased a Shield Beetle. Which means they shouldn't be facing Raiders without backup this early on. That being said, ten chasers is still going to be a fairly tough fight for most ships. Highly likely someone could get swarmed under fairly fast, but having it as chasers instead of a Raider means that people *can* pick off chasers then run away, and gradually wear down the opposition.

Reply:
> Because I want non-veteran players to be able to at least get to the point where they can buy basic Remnant ships without needing to have already purchased a Shield Beetle

Without passing judgment on whether or not this is the best way to go about it, I heartily approve of the goal.

Reply:
Raiders aren't actually all that damaging, so if you want to have the ability to "gradually wear down the opposition", a Raider without any means of repair or shield regeneration may be easier to deal with than 10 Chasers.

Reply:
Fair enough. Should I introduce a "worn down" raider variant that isn't nearly as bad of as the "Damaged" raider, but lacks certain key things contributing to their long-term durability? I'll be able to do that later on today.